MozillaFoundation/foundation.mozilla.org
# Returning localized signup in context instead of through .localized in the template

# Description
Related PRs/issues: [TP1-109](https://mozilla-hub.atlassian.net/browse/TP1-109)


This PR updates the `bannered_campaign_page.html` template by providing the localized sign up through the context, rather than using `.localized` within the template. 

# Steps to test:

1. Visit https://foundation-s-tp1-109-re-ft8c9b.mofostaging.net/en/campaigns/eu-misinformation/publisher-campaign/thanks-guardian/
2. Scroll down to the Sign Up Form, note that it is in English.
![Screenshot 2024-11-14 at 17-49-26 RA Thanks The Guardian - Mozilla Foundation](https://github.com/user-attachments/assets/d1a432eb-741d-4151-82ae-037bd5a9260c)

3. https://foundation-s-tp1-109-re-ft8c9b.mofostaging.net/es/campaigns/eu-misinformation/publisher-campaign/thanks-guardian/
4. Scroll down to the sign up form again, note that this time its rendering its translated version in Spanish.
![Screenshot 2024-11-14 at 17-50-07 RA Thanks The Guardian (IN SPANISH) - Fundación Mozilla](https://github.com/user-attachments/assets/340d2b0d-bd39-409d-a669-9bc98ac0aa11)
5. If everything is working as expected, testing is complete! 👍 




┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1485)
